src/sidebar/store/modules/test/annotations-test.js

```diff
@@ -236,8 +415,88 @@ describe('sidebar/store/modules/annotations', function() {
       );
     });
 
+    it('should set basic default properties on a new/empty annotation', () => {
+      store.dispatch(actions.createAnnotation({ id: 'myID' }, now));
+
+      const createdAnnotation = selectors.findAnnotationByID(
+        store.getState(),
+        'myID'
+      );
+
+      assert.equal(createdAnnotation.created, now.toISOString());
+      assert.equal(createdAnnotation.updated, now.toISOString());
+      assert.isArray(createdAnnotation.tags);
+      assert.isEmpty(createdAnnotation.tags);
+      assert.isString(createdAnnotation.text);
+      assert.isEmpty(createdAnnotation.text);
+    });
+
+    it('should set user properties on a new/empty annotation', () => {
+      store.dispatch(actions.createAnnotation({ id: 'myID' }, now));
+
+      const createdAnnotation = selectors.findAnnotationByID(
+        store.getState(),
+        'myID'
+      );
+
+      assert.equal(createdAnnotation.user, store.getState().session.userid);
+      assert.equal(
+        createdAnnotation.user_info,
+        store.getState().session.user_info
+      );
+    });
+
+    it('should set group to currently-focused group if not set on annotation', () => {
+      store.dispatch(actions.createAnnotation({ id: 'myID' }, now));
+
+      const createdAnnotation = selectors.findAnnotationByID(
+        store.getState(),
+        'myID'
+      );
+
+      assert.equal(
+        createdAnnotation.group,
+        store.getState().groups.focusedGroupId
+      );
+    });
+
+    it('should set not overwrite  properties if present', () => {
```

Comment:
```suggestion
    it('should set not overwrite properties if present', () => {
```

Remove an extra space.